<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeViewOptions</class>
 <widget class="QWidget" name="TreeViewOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Visible Columns</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_10">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
